### ISSUE TITLE
Hovmoeller: plot even if time series < 1year

### ISSUE THREAD

**Issue by georgmayr** [retostauffer/Rfoehnix#4]
For unusually short time series (but frequent enough foehn) it would be nice to also be able to plot a Hovmoeller and just have the times without data empty. Currently attempting to do so gives an empty plot.

**Comment by retostauffer** [comment]
Good point, `image(...)` now allows to set custom `xlim` and `ylim` values by providing a numeric vector of length `2L`.

* `ylim` restricted to `[0, 86400]` (seconds, time of the day)
* `xlim` restricted to `[0, 365]`. `365` is treated as `364` (leap year handling)
* if `xlim` values are decreasing, diagrams can be plotted across new years eve

Examples see: https://retostauffer.github.io/Rfoehnix/articles/image.html
